### ISSUE TITLE
Bug: SendFragment: Scan always clears Receivers address

### ISSUE THREAD

**Issue by jenniferberger** [m2049r/xmrwallet#124]
Preconditions:
Copy donation address

Steps:
1. Paste donation address into Receivers Address
2. Click on Scan Button
3. Click back

Expectation
The pasted address is still in the input

Actual
The pasted address is gone

**Comment by m2049r** [comment]
you mean if NO monero qr code is actually scanned?

**Comment by jenniferberger** [comment]
Yes, the address field is cleared when the user returns from the scan activity with the navigation button.
